### ISSUE TITLE
Uncaught TypeError: undefined is not a function (timeout)

### ISSUE THREAD

**Issue by rogerfar** [kendo-labs/angular-kendo#232]
I have a Kendo grid hooked with an editor template:

```
    $('#grid').kendoGrid({
        columns: [
            { field: "UserName", title: "Name", editor: self.employeeDropDownEditor },
        ],
    });
    self.employeeDropDownEditor = function (container, options) {
        $('<input required data-bind="value:' + options.field + '"/>')
            .appendTo(container)
            .kendoComboBox();
    };
```

Problem is that when I unfocus from the control when in edit mode I get this exception:
Line 757:

```
        timeout(function () {
            var scope = angular.element(self.element).scope();
            if (scope) {
                compile(self.element)(scope);
                digest(scope);
            }
        });
```

Where timeout is undefined.

Strange thing is that I have more grids where the function works fine, when I navigate to these grids (through routing), edit some kendo elements in the grid, navigate back to the example grid the timeout function is working.

I suspect a controller is loading up something but they are exactly equal, and anyway I don't think I should have to load up something manually except for "kendo.directives". 

So in a nutshell: the Angular timeout function is not always known to angular-kendo.
